generate-text-packs.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage()
+{
+	echo "Usage: $0 --name <package_name> --versions <package_versions> [options]"
+	echo ""
+	echo "  Restores specified package versions and generates a source-build-usable text-only package."
+	echo "  The resulting text-only packages can be found in the './src/textOnlyPackages/src' directory."
+	echo ""
+	echo "  Both --name and --versions must be specified"
+	echo ""
+	echo "  --name <package_name>          Name of the package. Example: Microsoft.Docker.Sdk (short: -n)"
+	echo "  --versions <package_versions>  Versions of the package, separated by a comma. Example: 1.1.0,1.0.0 (short: -v)"
+	echo ""
+	echo "Options:"
+	echo "  --feeds \"<value>\"            Semicolon-separated list of additional Nuget feeds to use during restore (short: -f)"
+}
+
+source="${BASH_SOURCE[0]}"
+
+# resolve $SOURCE until the file is no longer a symlink
+while [[ -h $source ]]; do
```

Comment:
Always use doublequotes in tests as otherwise the condition would become syntactically invalid when the variable would be empty.

Reply:
I recommend VS Code + Shellcheck:
https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck